.size-limit.js

```diff
@@ -7,7 +7,7 @@ module.exports = [
     name: 'Core - CDN',
     path: 'dist/legacy/rudder-analytics.min.js',
     gzip: true,
-    limit: '37.2 kB',
+    limit: '40.61 kB',
```

Comment:
this is too much! what have we added?

Reply:
only import was lodash.empty. `npm run check:size` is showing this size limit, Removed the lodash.empty import and reduced the size limit.